R/param_cal_method.R

```diff
@@ -0,0 +1,60 @@
+#' Methods for model calibration
+#' @param values A character string of possible values. See `values_cal_cls`
+#'  and `values_cal_reg` (in examples below).
+#'
+#' @details
+#' This parameter is used in postprocessing methods in the tailor package.
+#' @examples
+#' values_cal_cls
+#' class_cal_method()
+#'
+#' values_cal_reg
+#' class_reg_method()
+#' @rdname calibration
+#' @export
+class_cal_method <- function(values = values_cal_cls) {
+  new_qual_param(
+    type = "character",
+    values = values,
+    label = c(activation = "Calibration Method"),
```

Comment:
The name of the label still needs updating, same for the other parameter object

Reply:
I think that everyone, including me, has regretted my choice of a named vector here. 😄